ci/Dockerfile

```diff
@@ -36,7 +36,8 @@ ARG TEST_FOLDER="[none-if-not-testing]"
 
 # # RUN apk add --no-cache --virtual .build-deps gcc musl-dev rust cargo curl \
 RUN apk add --no-cache --virtual .build-deps cargo curl gcc musl-dev rust \
-    && curl -sSL https://install.python-poetry.org | python3 - \
+    && pip3 install --upgrade pip setuptools \
+    && curl -sSL https://install.python-poetry.org | POETRY_VERSION=1.8.4 python3 - \
```

Comment:
Pinning poetry version so we don't run into issues with pulling the latest.

Reply:
TY!